marlowe-dashboard-client/static/css/main.css

```diff
@@ -1,13 +1,16 @@
 @import url("https://fonts.googleapis.com/css2?family=Barlow:wght@400;600;700&display=swap");
 
+@import 'icons.css';
+@import 'tabs.css';
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
 html,
 body {
-  font-family: Barlow, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
+  font-family: Barlow, sans-serif;
+  min-width: 375px;
```

Comment:
not sure this is a good idea.... this will make smaller devices to have horizontal scrolling, right?